torchtune/config/_utils.py

```diff
@@ -148,3 +150,37 @@ def _merge_yaml_and_cli_args(yaml_args: Namespace, cli_args: List[str]) -> DictC
 
     # CLI takes precedence over yaml args
     return OmegaConf.merge(yaml_conf, cli_conf)
+
+
+def _get_template(template: str) -> PromptTemplate:
+    """
+    Get the prompt template class from the template string.
+
+    String should either be the PromptTemplate class name directly, or a raw
+    string with 1 or more placeholders. If none of these apply, then raise an
+    error.
+
+    Args:
+        template (str): class name of template, or string with placeholders
+
+    Returns:
+        PromptTemplate: the prompt template class or the same verified string
+
+    Raises:
+        ValueError: if the template is not a PromptTemplate class or a proper
+            template string
+    """
+    path = "torchtune.data." + template
```

Comment:
I'm confused, isn't this different from our usual instantiate logic? Why the change here?

Reply:
you know what I could just use instantiate

Reply:
no - this is strictly for the dataset builders. this method was originally in datasets/utils but I moved it to config since it was more akin to config functionality